Documentation/benchmarks/etcd-storage-memory-benchmark.md

```diff
@@ -0,0 +1,98 @@
+# Storage Memory Usage Benchmark
+
+<!---todo: link storage to storage design doc-->
+The storage has two components that consume physical memory: the _in-memory index_, and the _page cache_ for on-disk data.
```

Comment:
Two components of etcd storage consume physical memory. The etcd process allocates an *in-memory index* to speed key lookup. The process's *page cache*, managed by the operating system, stores recently-accessed data from disk for quick re-use.